deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -0,0 +1,114 @@
+id: hawkular-cassandra-node-dynamic-pv
+kind: Template
+apiVersion: v1
+name: 'Hawkular Cassandra Node Template: Persistent Storage'
```

Comment:
I see we are following the pattern here, but @mwringe wouldnt we rather utilize a shorter name that does not need to be enclosed in quotes when we reference it?